src/electron/index.ts

```diff
@@ -130,6 +136,55 @@ app.on('ready', () => {
   })
   mainWindow = createMainWindow()
 
+  ipcMain.on('menuAcceleratorChanged', (_, args) => {
+    if (args.length != 2) {
+      return
+    }
+    const menuItemId = args[0]
+    const newAcceleratorShortcut =
+      args[1] == null
+        ? undefined
+        : MAC
+        ? args[1].replace('Ctrl', 'Cmd')
+        : args[1]
+
+    function updateTemplate(
```

Comment:
Using position to fix the template is not a good idea in my opinion. We need some other map.

The template should be the only one that decide position and the map should decide which key to bind. 
Otherwise, we have to deal with the position every time we add/remove menus.

Reply:
Will refactor to map, update the map and then updateMenu